File: app/src/main/java/cn/chenyuanming/gankmeizhi/beans/db/FavoriteBean.java
package cn.chenyuanming.gankmeizhi.beans.db;

import com.litesuits.orm.db.annotation.PrimaryKey;
import com.litesuits.orm.db.enums.AssignType;

import java.util.TreeSet;

/**
 * Created by Administrator on 2016/1/30.
 */
public class FavoriteBean {

    @PrimaryKey(AssignType.BY_MYSELF)
    public int id;
    public TreeSet<String> favorites = new TreeSet<>();
}

File: app/src/main/java/cn/chenyuanming/gankmeizhi/beans/db/FavoriteBean.kt
package cn.chenyuanming.gankmeizhi.beans.db

import com.litesuits.orm.db.annotation.PrimaryKey
import com.litesuits.orm.db.enums.AssignType
import java.util.*

/**
 * Created by Chen Yuanming on 2016/1/30.
 */
class FavoriteBean {

    @PrimaryKey(AssignType.BY_MYSELF)
    var id: Int = 0
    var favorites = TreeSet<String>()
}
